concurrent_test.go

```diff
@@ -60,12 +71,14 @@ func TestConcurrentReadAndWrite(t *testing.T) {
 	if testing.Short() {
 		t.Skip("skipping test in short mode.")
 	}
-	bucket := []byte("data")
-	keys := []string{"key0", "key1", "key2", "key3", "key4", "key5", "key6", "key7", "key8", "key9"}
+
+	testDuration := concurrentTestDuration(t)
 	conf := concurrentConfig{
+		bucketCount: 5,
+		keyCount:    10000,
 		workInterval: duration{
-			min: 5 * time.Millisecond,
-			max: 10 * time.Millisecond,
+			min: 0 * time.Millisecond,
```

Comment:
Just not set `workInterval`  as lack of delay can be treated as default behavior.

Reply:
Changed to` workInterval: duration{},`.

I want to explicitly set each configuration item, even it's zero value, to indicate that its a conscious choice instead of missing.